doc/manual/source/command-ref/nix-instantiate.md

```diff
@@ -205,3 +210,10 @@ $ nix-instantiate --eval --xml --strict --expr '{ x = {}; }'
   </attrs>
 </expr>
 ```
+
+Replacing evaluation errors:
+
+```console
+$ nix-instantiate --eval --json --replace-eval-errors --expr '{ a = throw "fail"; }'
+{"a":"«evaluation error»"}
```

Comment:
Note that this representation of eval errors is ambiguous, since
```
nix-instantiate --eval --json --expr '{ a = "«evaluation error»"; }'
{"a":"«evaluation error»"}
```
produces the same output.

Reply:
While this is true, we already have such ambiguity, for instance:
```
nix-instantiate --eval --json --expr --strict '{ foo.outPath = "bar"; }'
{"foo":"bar"}
```

That said, something like `{"__nix_evaluation_error": null}` would be subjectively less ambiguous, and allows for a message or other data to be put where it says `null`.

Considering that this originated from a `tryEval` improvement, we don't have to be so afraid of the "chaos" in the message anymore!